CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 8.0.0-alpha.1
+
+- The first public preview of our [new high-performance core API and extensions](https://github.com/App-vNext/Polly/issues/1048) - Thanks to:
+  - [@andrey-noskov](https://github.com/andrey-noskov)
+  - [@joelhulen](https://github.com/joelhulen)
+  - [@juraj-blazek](https://github.com/juraj-blazek)
+  - [@geeknoid](https://github.com/geeknoid)
+  - [@martincostello](https://github.com/martincostello)
+  - [@martintmk](https://github.com/martintmk)
+  - [@SimonCropp](https://github.com/SimonCropp)
+  - [@terrajobst](https://github.com/terrajobst)
```

Comment:
Let me know if I forgot anyone!

Reply:
Let's also mention:

- @tekian: he helper me shape the API surface
- @laura-mi: she was influential behind the resilience API that was precursor to V8
- @adamnova: he provided great internal feedback